prow/cmd/deck/tide.go

```diff
@@ -52,6 +53,9 @@ type tideAgent struct {
 	hiddenOnly  bool
 	showHidden  bool
 
+	tenantIDs []string
+	cfg       config.Config
```

Comment:
This should be a `config.Getter` function rather than a `config.Config` struct so that we don't have to restart Deck in order for it to pick up config changes.